frontend/src/components/PlantDetailsView/PlantDetailsView.tsx

```diff
@@ -52,7 +52,7 @@ const PlantDetailsView: React.FC<{}> = (props) => {
                 <Row>
                     <Col xxl={6}>
                         <Card as={DetailsWrapper}>
-                            <Card.Img variant="top" src={plant.photo} />
+                            <Card.Img variant="top" src={base64Header + plant.photo} />
```

Comment:
dobrze, że jest to tak doklejane, wtedy w bazie trzymamy samo zakodowane zdjęcie